docs/docs/reactions/index.md

```diff
@@ -106,6 +106,8 @@ const source = MessageDbSource.create({
   tailSleepIntervalMs: 100, 
   // How many streams are we OK to process concurrently?
   maxConcurrentStreams: 10, 
+  // How many batches can be pre-loaded beyond what's been completed (checkpointing is always sequential in order of age)
```

Comment:
need to convey that it includes the actual batch being processed. the concurrent batches name was perhaps better!